packages/icons/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@packages/icons",
+  "version": "0.0.0-development",
+  "description": "Cypress Icons",
+  "main": "index.js",
+  "scripts": {
+    "build": "scripts/build.sh",
+    "postbuild": "ts-node ./src/ico.ts",
+    "test": "NODE_ENV=test mocha",
+    "test-watch": "NODE_ENV=test mocha --watch",
+    "prepublish": "npm run build",
+    "release": "releaser"
+  },
+  "devDependencies": {
+    "@types/mocha": "^9.1.0",
```

Comment:
Nit, can we use 8.x types for mocha - this is what most of our other packages reference. Can we match the types version?

Reply:
@elevatebart can we change this one?

Reply:
you got it !!